### DISCUSSION TITLE
How to update?

### DISCUSSION THREAD

**Discussion by Finkregh** [gcormier/megadesk#106] [Q&A] [answered]
Hi,

perhaps it is too warm here, but i did not find a walkthrough on how to flash an updated firmware to the controller.
Did i miss something or could you point me somewhere?

Thanks!

**Comment by gcormier** [comment, marked as answer]
https://github.com/gcormier/megadesk/blob/master/DIY.md

Does this help out? There are also sample flash batch/shell files. Let me know if you have any questions.

**Reply by Finkregh** [reply]
Thanks, i hoped for something more "you are doing this for the first time", perhaps 'you'll get a PR then :)

**Comment by gcormier** [comment]
If it's regarding flashing an AVR/Arduino in general, there will be far more comprehensive tutorials out there then I can provide :)
